notebooks/official/experiments/build_model_experimentation_lineage_with_prebuild_code.ipynb

```diff
@@ -0,0 +1,1229 @@
+{
```

Comment:
<p>move pip installs into the cell above (single cell for installs)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/637/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>done</p>